src/LuaDebuggerLauncher/RuleProperties.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright © Microsoft Corporation.  All Rights Reserved.
+// This code released under the terms of the 
+// Microsoft Public License (MS-PL, http://opensource.org/licenses/ms-pl.html.)
+
+namespace Microsoft.VisualStudio.Debugger.Lua.Rules
+{
+    using System.ComponentModel.Composition;
+    using Microsoft.VisualStudio.ProjectSystem;
+    using Microsoft.VisualStudio.ProjectSystem.Utilities;
+    using Microsoft.VisualStudio.ProjectSystem.Properties;
+
+    /// <summary>
+    /// Provides rule-based property access.
+    /// </summary>
+    [Export]
+    [AppliesTo(ProjectCapabilities.VisualC)]
+    internal partial class RuleProperties : StronglyTypedPropertyAccess
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RuleProperties"/> class.
+        /// </summary>
+        [ImportingConstructor]
+        public RuleProperties([Import(RequiredCreationPolicy = CreationPolicy.Shared)] ConfiguredProject configuredProject)
```

Comment:
The `[Import]` attribute shouldn't be necessary. Especially with the RequiredCreationPolicy specified.Can you remove it to improve readability of the method signature?